maintainers/maintainer-list.nix

```diff
@@ -5524,6 +5524,12 @@
     githubId = 7622248;
     name = "Sebastian Zivota";
   };
+  locallycompact = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add locallycompact``.